xapian-core/diversify/Makefile.mk

```diff
@@ -0,0 +1,3 @@
+lib_src +=\
+	diversify/diversify.cc
+
```

Comment:
It's preferable to avoid extra blank lines at the end of files (unless the file format requires them, which isn't the case here).

Reply:
Fixed in a new commit.